src/NzbDrone.Core/Indexers/Definitions/DesiTorrents.cs

```diff
@@ -19,11 +21,6 @@ public DesiTorrents(IIndexerHttpClient httpClient, IEventAggregator eventAggrega
         {
         }
 
-        public override IParseIndexerResponse GetParser()
-        {
-            return new DesiTorrentsParser(Settings, Capabilities);
```

Comment:
Think you need to keep this part as well, or it will just use the base United Indexer